string input_char
---
bool response_message
